src/components/BookForm.js

```diff
@@ -1,13 +1,20 @@
 import { PropTypes } from 'prop-types';
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { add, remove } from '../redux/books/booksSlice';
 
-function Books({ title, author }) {
+function Books({ title, author, id }) {
```

Comment:
- [ ] It will be great to separate the components inside your bookForm.js It seems you have created different components and placed them in just one file. For example, this component is responsible for rendering the books. It should be on a separate file called maybe book.js.

-   The bookForm file should only have one component which is the bookForm component that is responsible for rendering the form.